.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "jsonapi-kotlin"]
+	path = jsonapi-kotlin
+	url = git@github.com:phongvhd93/nimble-jsonapi-kotlin.git
```

Comment:
Let's use https://github.com/nimblehq/jsonapi-kotlin, if there is issue with minimum version, we can update our project instead

Reply:
@suho, I solved in ead4a4a